packages/backend/src/core/MfmService.ts

```diff
@@ -171,6 +171,41 @@ export class MfmService {
 					break;
 				}
 
+				case 'ruby': {
+					const ruby: [string, string][] = [];
+					for (const child of node.childNodes) {
+						if (child.nodeName === 'rp') {
+							continue;
+						}
+						if (treeAdapter.isTextNode(child) && !child.value.match(/\s|\[|\]/)) {
+							ruby.push([child.value, '']);
+							continue;
+						}
+						if (child.nodeName === 'rt' && ruby.length > 0) {
+							const rt = getText(child);
+							if (rt.match(/\s|\[|\]/)) {
+								// If any space is included in rt, it is treated as a normal text
+								break;
+							} else {
+								ruby[ruby.length - 1][1] = rt;
```

Comment:
```suggestion
								ruby.at(-1)![1] = rt;
```

The indexed access isn't actually type-checked (https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#checked-indexed-accesses-(--nouncheckedindexedaccess)), so this is no less type-safe even though it adds `!`.